app/views/users/index.html.erb

```diff
@@ -1,54 +1,42 @@
 <% provide :title, 'Members' %>
 <div class="row">
-  <div class="col-xs-12 col-sm-8">
-    <h1>All members</h1>
-    <% if @users.empty? %>
+  <div class="col-xs-12 col-sm-12">
+    <h1>Members Directory</h1>
+    <% if @users_count == 0 %>
         <p>It is a lonely planet we live in&#8230;</p>
     <% else %>
-        <p>Check out our <%= @users.count %> awesome volunteers from all over the globe!</p>
+        <p>Check out our <%= @users_count %> awesome volunteers from all over the globe!</p>
     <% end %>
   </div>
-  <div class="col-sm-4">
+  <% if @current_user %>
+    <div class="col-sm-12 well">
+      <p>Filter users by</p>
+      <%= form_tag(users_path, remote: true, method: "get", class: "form-inline filters-users-advanced") do %>
+        <%= select_tag :project_filter, options_from_collection_for_select(@projects, 'id', 'title', params[:project_filter]), {  prompt: 'Project involvement...' } %>
+        <%= select_tag :timezone_filter, options_for_select(['Close To My Timezone Area', 'Wider Timezone Area'], params[:timezone_filter]), { prompt: 'Time zone...' } %>
+        <%= check_box_tag :online, params[:online], (true if !params[:online].nil?) %> Recently Online
+        <%= submit_tag 'submit' %>
+      <% end %>
+    </div>
+  <% end %>
+</div>
+<div class="row">
+  <div class="col-sm-12">
     <form class="form-inline users-filter" role="form">
-      <div class="form-group">
+      <div class="form-group inner-addon left-addon">
         <label class="sr-only" for="user-filter">Filter users</label>
-        <input type="text" class="form-control" id="user-filter" placeholder="filter users..">
+        <i class="fa fa-search"></i>
+        <input type="text" class="form-control" id="user-filter" placeholder="Search members...">
       </div>
     </form>
   </div>
 </div>
 <div class="row">
-  <% unless @users.empty? %>
+  <div class="col-sm-12">
+    <% unless @users.empty? %>
       <ul class="media-list row" id="UsersList">
```

Comment:
Although there isn't a standard naming convention for html/css ids, I would set this one to `user-list` The most common naming standards are:
There are several options:
```
id="myId"
id="my-id"
id="my_id"
```
From all of these I think that the dash look a lot cleaner than under_scores or camelCase.